### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by Bkk20** [benleb/surepy#78] [Q&A]
Hi Ben,
Sorry but I am not so in programming but I wanna use your great project.

Do I need the Pet connect hardware and also the app to use your project? I dont know from where I get the API.

**Comment by benleb** [comment]
Hey, surepy can be seen as an alternative to the iOS/Android app. You need, of course, the hardware, meaning a flap, feeder or felaqua and their hub.
To access their API you will also have to give surepy a Sure Petcare account. This is normale created in the app but can also be done at surehub.io

If you want to use such hardware with Home Assistant, there is a  integration for it built-in, you won't need to use this library and code something yourself then.
